tests/component_flask_rest/tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+setupdir = {toxinidir}/../..
+envlist =
+    py27,py35,py36,py37,py38,pypy,pypy3
+
+[pytest]
+usefixtures = session_initialization requires_data_collector
+
+[testenv]
+deps =
+    flask<0.13
+    flask-restful
+    flask-restplus
+    flask-restx
+
+setenv =
+    PYTHONPATH={toxinidir}/..
+    TOX_ENVDIR = {envdir}
+    without-extensions: NEW_RELIC_EXTENSIONS = false
```

Comment:
Since we aren't testing with and without extensions explicitly in our matrix, we can probably remove this.